components/credit-balance-card.tsx

```diff
@@ -47,9 +45,10 @@ export function CreditBalanceCard({ handleAddFunds }: { handleAddFunds: () => vo
         </div>
         <Button
           className="w-full bg-primary hover:bg-secondary-foreground text-base"
-          onClick={() => (loggedIn ? handleAddFunds : updateState({ showLogin: true }))}
+          disabled={!loggedIn}
+          onClick={handleAddFunds}
```

Comment:
Looks like this whole file was not rebased, just forced pushed.
Revert all changes in this file.